tests/unit/phpDocumentor/Guides/DocumentTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * phpDocumentor
+ *
+ * PHP Version 5.5
+ *
+ * @copyright 2010-2015 Mike van Riel / Naenius (http://www.naenius.com)
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\Guides;
+
+use phpDocumentor\Path;
+
+/**
+ * Test case for Document
+ * @coversDefaultClass phpDocumentor\Guides\Document
+ */
+class DocumentTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @covers ::__construct
+     * @covers ::getTitle
+     * @covers ::getPath
+     * @covers ::getContent
+     */
+    public function testGetters()
+    {
+        $document = new DummyDocument(new Path('my/Path'), 'SomeTitle', 'MyContent');
```

Comment:
I know the partial mocking technique. But I think it is slower than just creating a dummy / stub. And since I'm running the test suite very often. (Partial and Full) I would like to keep things fast.